SlackPirate.py

```diff
@@ -153,12 +153,17 @@ def display_cookie_tokens(cookie):
         r = requests.get("https://slackpirate-donotuse.slack.com", cookies=cookie)
         already_signed_in_match = re.findall(ALREADY_SIGNED_IN_TEAM_REGEX, str(r.content))
         if already_signed_in_match:
+            selected_agent = getUserAgent()
```

Comment:
I think this is redundant as this variable is already set during the initialization of the script.

Reply:
That selected_agent isn't passed to display_cookie_tokens(), so it has no idea that that variable already exists. The selected_agent has to be created inside display_cookie_tokens(), because the `collected_output_info = init_scanning_context(token=slack_token, user_agent=selected_agent)` line needs the user_agent for each token's scanning context init.
Althought there may be a better way to do this.

Reply:
Perhaps add a parameter to `display_cookie_tokens` for the user agent string. That would allow re-use of the one that was selected already 🤷‍♂️

Reply:
Duh, didn't think of that for some reason. Added it just now.